tests/vnc_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				vncConnect()
 			})
 
-			It("should allow accessing the VNC device multiple times", func() {
+			It("[test_id:2925]should allow accessing the VNC device multiple times", func() {
```

Comment:
Right, that's not so clear indeed.
Before the latest change in regards to console connections, it was possible to connect to the VMI through VNC console multiple times.
After that change, when you try to connect via a second connection the first (existing) connection would be stopped and the second connection is established.
2925 should actually mean that and state that it's ok for the first connection to be terminated (it's not a requirement, it's a consequence) - the most important thing to check is that the second connection is established

Reply:
I certainly agree. In this case I would like to have test case updated to reflect the reality.
This way we make sure, it will not return as not implemented in the future.
@dshchedr do you agree?

Reply:
As I see, current test includes the previous one [0], I'm not sure we really need to have these duplicates, so what I would do:

1) remove test where we trying to connect to vnc only once [0]
2) mark this test as [test_id:1611]
3) update testcase [1] in polarion with additional step:  repeat several times 

[0] https://github.com/kubevirt/kubevirt/blob/5e30746aa0a8d71744c4b9de24f07a99d99ced14/tests/vnc_test.go#L113

[1] https://polarion.engineering.redhat.com/polarion/redirect/project/CNV/workitem?id=CNV-1611

Reply:
This works for me. This way it will be clear.